Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -15,6 +15,14 @@
 f = open(os.path.join(vcs.prefix,"share","vcs","wmo_symbols.json"))
 wmo = json.load(f)
 
+def isTrue(x):
+    """Test if an unsanitized user input is 'true'."""
+    # I'm just encapsulating existing logic here, but this seems to miss some
+    # other cases users may enter. What about:
+    #    - Non-zero integers other than 1?
+    #    - "yes", "Yes", "Y"?
+    return x in ("y", 1, True)
```

Comment:
Let's not touch these functions per say, we might need these for something else that really just need yesno (and not boolean) But you're right let's create an issue to create a unifying function that understands all these and sets to True/False, checkBoolean should really only check if the input is a boolean, let's think of a better name, I liked checkOnOff but that's already used.Maybe checkTrigger ?